### PR TITLE
Address some Symfony upgrade leftovers

### PR DESCRIPTION
**Pull request by MKodde**
Each commit describes what changed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,7 +13,6 @@
 /vendor/
 /web/bundles/
 /.idea
-.env
```
Comment (MKodde) [inline .gitignore:16]: We do not use `.env` files, but use the more traditional `parameters.yaml` way of configuring app parameters.

Comment (johanib) [inline .gitignore:16]: Why remove this? Cant hurt to leave it there and prevent potential credential leaks?

File: config/services.yaml [hunk 2]
```diff
@@ -8,6 +8,8 @@ imports:
 
 services:
     _defaults:
+        bind:
+            $isDeprovisionApi: '%deprovision_api_settings_enabled%'
```
Comment (johanib) [inline config/services.yaml:12]: This is convention or configured. I assume this looks at the feature toggle below :)

File: composer.lock [hunk 3]
```diff
@@ -271,16 +271,16 @@
         },
         {
             "name": "doctrine/dbal",
-            "version": "4.0.2",
+            "version": "4.1.0",
```
Comment (johanib) [inline composer.lock:274]: Why not adopt `composer bump` to make your reviewer's live easier? ;)

**Review by johanib — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (MKodde) [reply .gitignore:16]: We could leave it, but makes little sense if we do not eval/use them.

**Pull request merged**